### PR TITLE
CU-86druab11 - NEON3 - Wallets Screen - Change implementation from side bar into the new screen under "Connections" tab

### PR DESCRIPTION
**Pull request by Thacryba**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by melanke** [comment]
Task linked: [CU-86druab11 NEON3 - Wallets Screen - Change implementation from side bar into the new screen under "Connections" tab](https://app.clickup.com/t/86druab11)

File: src/renderer/src/helpers/DateHelper.ts [hunk 1]
```diff
@@ -8,4 +8,8 @@ export class DateHelper {
     const date = new Date(unixTime * 1000)
     return date.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' })
   }
+
+  static timeToDateTime = (unixTime: number): string => {
```
Comment (raulduartep) [inline src/renderer/src/helpers/DateHelper.ts]: Rename it to a more descriptive name, for example, unixToDateHour

File: src/renderer/src/routes/pages/Wallets/index.tsx [hunk 2]
```diff
@@ -209,12 +206,7 @@ export const WalletsPage = () => {
               <SidebarMenuButton title={t('tokens')} to={`/wallets/${selectedAccount?.address}/tokens`} />
               <SidebarMenuButton title={t('nfts')} to={`/wallets/${selectedAccount?.address}/nfts`} />
               <SidebarMenuButton title={t('transactions')} to={`/wallets/${selectedAccount?.address}/transactions`} />
-              <SidebarMenuButton
-                title={t('connections')}
-                type="button"
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Wallets/index.tsx:166]: You won't need this prop anymore, so Rename the component to `<SidebarMenuLink />` and remove this prop and the condicional from the component to always render a link

File: src/renderer/src/routes/pages/Connections/ConnectionsTable.tsx
Comment (raulduartep) [inline src/renderer/src/routes/pages/Connections/ConnectionsTable.tsx:1]: Move it to components folder

File: src/renderer/src/routes/pages/Connections/ConnectionsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+type TConnectionsTableProps = {
+  sessions: TSession[]
+  hasAddress?: boolean
+}
+
+const columnHelper = createColumnHelper<TSession>()
+
+export const ConnectionsTable = forwardRef<HTMLDivElement, TConnectionsTableProps>(
+  ({ sessions, hasAddress = false }, ref) => {
+    const { t } = useTranslation('pages', { keyPrefix: 'connections.table' })
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Connections/ConnectionsTable.tsx]: Change the namespace to components instead of pages

**Pull request merged**